static/app/components/replays/table/deleteReplays.tsx

```diff
@@ -35,6 +37,7 @@ interface Props {
 }
 
 export default function DeleteReplays({selectedIds, replays, queryOptions}: Props) {
+  const analyticsArea = useAnalyticsArea();
```

Comment:
i replaced the original referrer string with `analyticsArea` so that the two files in this pr have the same value. They both live inside the `replay.list` area.